aggregate.go

```diff
@@ -118,6 +160,7 @@ func (a *Aggregator) containsAggregatedBundle(list [][]Bundle, item []Bundle) bo
 func (a *Aggregator) containsBundleByName(list []Bundle, item Bundle) bool {
 	for _, b := range list {
 		if b.Name == item.Name {
+			a.logger.Log("level", "debug", "message", "version bundle already exists in aggregated list", "name", item.Name)
```

Comment:
The "aggregated list" is the list of version bundles which actually are supposed to make up a release. Not sure if we should just use "release" as wording here.